tools/generate-countries.ipynb

```diff
@@ -290,12 +311,37 @@
     "\n",
     "    modify_template(templatename, os.path.join(\"ipynb\", output_file_name), mappings, wwwroot)\n",
     "    assert os.path.exists(output_file_path), f\"{output_file_path} does not exist\"\n",
-    "    return output_file_name\n"
+    "    return output_file_name\n",
+    "\n",
+    "def create_ipynb_for_hungary(region, subregion='', templatename='', wwwroot=''):\n",
+    "    \"\"\"Creates ipynb file for region and subregion in Germany, based on templatename. \n",
+    "    File is based in ipynb subfolder of wwwroot.\n",
+    "    Returns name of file.\n",
+    "    \n",
+    "    ===== Region, subregion or county should be used.  Megnézni hol van ilyesmire hivatkozás és ha nincs akkor kitörölni.\n",
+    "    \"\"\"\n",
+    "    hungary_check_region_name_is_known(region)\n",
+    "    \n",
+    "    output_file_name =  f\"Hungary-{sanitise(region)}.ipynb\"\n",
+    "    output_file_path = os.path.join(wwwroot, \"ipynb\", output_file_name)\n",
+    "    \n",
+    "    mappings = {\n",
+    "        \"%title%\" : f\"Hungary: {region}\",\n",
+    "        \"%title2%\" : \"\",\n",
+    "        \"%region%\" : region,\n",
+    "        \"%subregion%\" : \"\",\n",
```

Comment:
```suggestion
```

Is the subregion argument needed? If it's always empty might as well remove it.

Reply:
It is not used. Ok